presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestGeoFunctions.java

```diff
@@ -923,6 +923,28 @@ private void assertSTGeometryN(String wkt, int index, String expected)
         assertFunction("ST_ASText(ST_GeometryN(ST_GeometryFromText('" + wkt + "')," + index + "))", VARCHAR, expected);
     }
 
+    @Test
+    public void testSTLineString()
+    {
+        // General case, 2+ points
+        assertFunction("ST_LineString(array[ST_Point(1,2), ST_Point(3,4)])", GEOMETRY, "LINESTRING (1 2, 3 4)");
+        assertFunction("ST_LineString(array[ST_Point(1,2), ST_Point(3,4), ST_Point(5, 6)])", GEOMETRY, "LINESTRING (1 2, 3 4, 5 6)");
+        assertFunction("ST_LineString(array[ST_Point(1,2), ST_Point(3,4), ST_Point(5,6), ST_Point(7,8)])", GEOMETRY, "LINESTRING (1 2, 3 4, 5 6, 7 8)");
+        // Other ways of creating points
+        assertFunction("ST_LineString(array[ST_GeometryFromText('POINT (1 2)'), ST_GeometryFromText('POINT (3 4)')])", GEOMETRY, "LINESTRING (1 2, 3 4)");
+
+        // Single point
+        assertInvalidFunction("ST_LineString(array[ST_Point(9,10)])", INVALID_FUNCTION_ARGUMENT, "LineString cannot have one point");
+        assertInvalidFunction("ST_LineString(array[ST_GeometryFromText('POINT (1 2)')])", INVALID_FUNCTION_ARGUMENT, "LineString cannot have one point");
+
+        // Zero points
+        assertFunction("ST_LineString(array[])", GEOMETRY, "LINESTRING EMPTY");
+
+        // Invalid points
+        assertInvalidFunction("ST_LineString(array[ST_POINT(7,8), ST_GeometryFromText('LINESTRING (1 2, 3 4)')])", INVALID_FUNCTION_ARGUMENT, "ST_LineString takes only an array of valid points, LineString was passed");
+        assertInvalidFunction("ST_LineString(array[ST_POINT(7,8), ST_GeometryFromText('LINESTRING (1, 3 4)')])", INVALID_FUNCTION_ARGUMENT, "Invalid WKT: LINESTRING (1, 3 4)");
```

Comment:
This tests verifies the behavior of `ST_GeometryFromText`, which is not relevant here. Could you test empty points instead?